packages/browser/yarn.lock

```diff
@@ -1856,6 +1856,14 @@
     "@types/prop-types" "*"
     csstype "^2.2.0"
 
+"@types/react@^16.x":
```

Comment:
I am not sure if we should keep yarn.lock file in git repo.

Reply:
> the yarn.lock file will ensure that they get precisely the same dependencies as you have.

We should keep yarn.lock file in git repo.